.gitignore

```diff
@@ -34,6 +34,9 @@ jspm_packages
 # Optional npm cache directory
 .npm
 
+# npm package lock file
+package-lock.json
```

Comment:
This doesn't hurt anything, but FYI we now use yarn instead of npm to manage packages.